<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>User</class>
 <widget class="QDialog" name="User">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>-10</x>
     <y>-11</y>
     <width>811</width>
     <height>91</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#frame{background:#00204a}</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QLabel" name="username_label">
    <property name="geometry">
     <rect>
      <x>99</x>
      <y>34</y>
      <width>66</width>
      <height>37</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Kalameh ExtraBold</family>
      <pointsize>13</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel#username_label{
color:white;
}</string>
    </property>
    <property name="text">
     <string>User</string>
    </property>
   </widget>
   <widget class="QLabel" name="profilephoto_label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>18</y>
      <width>64</width>
      <height>64</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="assets.qrc">:/icons/D:/grocery_store/grocery_store/assets/icons/profile.png</pixmap>
    </property>
   </widget>
   <widget class="QPushButton" name="editprofile_button">
    <property name="geometry">
     <rect>
      <x>176</x>
      <y>40</y>
      <width>31</width>
      <height>25</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton#editprofile_button{
background:#00204a;
border-radius:30px;
border: 0px solid #005792;
}
</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>D:/grocery_store/grocery_store/assets/icons/pencil.png</normaloff>D:/grocery_store/grocery_store/assets/icons/pencil.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>15</width>
      <height>15</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="cart">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>20</y>
      <width>51</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#cart{
background:#00204A;
border:noun;}</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>D:/grocery_store/grocery_store/assets/icons/shopping-cart.png</normaloff>D:/grocery_store/grocery_store/assets/icons/shopping-cart.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>61</width>
      <height>49</height>
     </size>
    </property>
   </widget>
   <widget class="QLabel" name="cart_count">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>33</y>
      <width>21</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Kalameh ExtraBold</family>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">#cart_count{
background:rgb(170, 0, 0);
border-radius:5px;
color:white;}</string>
    </property>
    <property name="text">
     <string>0</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="exit">
    <property name="geometry">
     <rect>
      <x>742</x>
      <y>32</y>
      <width>61</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#exit{
background:#00204A;
border:noun}</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>D:/grocery_store/grocery_store/assets/icons/logout.png</normaloff>D:/grocery_store/grocery_store/assets/icons/logout.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>30</width>
      <height>30</height>
     </size>
    </property>
   </widget>
  </widget>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>4</x>
     <y>89</y>
     <width>791</width>
     <height>501</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Kalameh Bold</family>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="currentIndex">
    <number>1</number>
   </property>
   <property name="tabsClosable">
    <bool>true</bool>
   </property>
   <widget class="QWidget" name="Home">
    <attribute name="title">
     <string>Home</string>
    </attribute>
    <widget class="QFrame" name="frame_2">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>120</y>
       <width>415</width>
       <height>332</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">#frame_2{
background:#00bbf0;
border-radius:25px;
color:#d9faff}</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QTableView" name="product_list">
      <property name="geometry">
       <rect>
        <x>13</x>
        <y>18</y>
        <width>389</width>
        <height>298</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Kalameh Bold</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">#product_list{
background:#00bbf0;
border:noun;
color:#00204a}</string>
      </property>
      <attribute name="horizontalHeaderMinimumSectionSize">
       <number>800</number>
      </attribute>
      <attribute name="horizontalHeaderDefaultSectionSize">
       <number>800</number>
      </attribute>
     </widget>
    </widget>
    <widget class="QLabel" name="deleteproduct_lable_4">
     <property name="geometry">
      <rect>
       <x>470</x>
       <y>120</y>
       <width>211</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Kalameh Medium</family>
       <pointsize>9</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel{
color:#005792}</string>
     </property>
     <property name="text">
      <string>product name</string>
     </property>
    </widget>
    <widget class="QFrame" name="frame_11">
     <property name="geometry">
      <rect>
       <x>460</x>
       <y>360</y>
       <width>101</width>
       <height>51</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">#frame_11{
background:#00bbf0;
border-radius:25px;
color:#00204a;
}</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QLabel" name="price_2">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>10</y>
        <width>261</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Kalameh ExtraBold</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QLabel{
color:#005792}</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </widget>
    <widget class="QComboBox" name="group">
     <property name="geometry">
      <rect>
       <x>360</x>
       <y>40</y>
       <width>181</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Kalameh Bold</family>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">#group
{
background:#00bbf0;
border-radius:25px;
color:#00204a;
}</string>
     </property>
    </widget>
    <widget class="QLabel" name="name_4">
     <property name="geometry">
      <rect>
       <x>370</x>
       <y>10</y>
       <width>101</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Kalameh ExtraBold</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel{
color:#005792}</string>
     </property>
     <property name="text">
      <string>group</string>
     </property>
    </widget>
    <widget class="QFrame" name="frame_6">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>40</y>
       <width>311</width>
       <height>51</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">#frame_6{
background:#00bbf0;
border-radius:25px;
color:#00204a;
}</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QPushButton" name="searchButton">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>10</y>
        <width>51</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">#searchButton{
background:#00bbf0;
border-radius:25px;
color:#00204a;
}</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>../../Downloads/magnifying-glass.png</normaloff>../../Downloads/magnifying-glass.png</iconset>
      </property>
     </widget>
    </widget>
    <widget class="QFrame" name="frame_10">
     <property name="geometry">
      <rect>
       <x>460</x>
       <y>260</y>
       <width>311</width>
       <height>51</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">#frame_10{
background:#00bbf0;
border-radius:25px;
color:#00204a;
}</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QLabel" name="price">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>10</y>
        <width>261</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Kalameh ExtraBold</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QLabel{
color:#005792}</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </widget>
    <widget class="QPushButton" name="addtocart">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>580</x>
       <y>360</y>
       <width>191</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Kalameh Bold</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">#addtocart{
background:#005792;
border-radius:25px;
color:#d9faff;

}
#addtocart:hover{
background:#00204a;
border-radius:25px;
color:#d9faff;

}</string>
     </property>
     <property name="text">
      <string>Add to cart</string>
     </property>
    </widget>
    <widget class="QLabel" name="deleteproduct_lable_5">
     <property name="geometry">
      <rect>
       <x>470</x>
       <y>230</y>
       <width>211</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Kalameh Medium</family>
       <pointsize>9</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel{
color:#005792}</string>
     </property>
     <property name="text">
      <string>price</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="count">
     <property name="geometry">
      <rect>
       <x>470</x>
       <y>360</y>
       <width>81</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Kalameh Bold</family>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">#count{

background:#00bbf0;
border-radius:25px;
color:#00204a;
}</string>
     </property>
     <property name="minimum">
      <number>1</number>
     </property>
    </widget>
    <widget class="QFrame" name="frame_7">
     <property name="geometry">
      <rect>
       <x>350</x>
       <y>40</y>
       <width>191</width>
       <height>51</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">#frame_7
{
background:#00bbf0;
border-radius:25px;
color:#00204a;
}</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
    </widget>
    <widget class="QFrame" name="frame_9">
     <property name="geometry">
      <rect>
       <x>460</x>
       <y>150</y>
       <width>311</width>
       <height>51</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">#frame_9{
background:#00bbf0;
border-radius:25px;
color:#00204a;
}</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QLabel" name="productname">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>9</y>
        <width>261</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Kalameh ExtraBold</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QLabel{
color:#005792}</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </widget>
    <widget class="QLabel" name="search_lable">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>10</y>
       <width>101</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Kalameh ExtraBold</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel{
color:#005792}</string>
     </property>
     <property name="text">
      <string>search</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="available">
     <property name="geometry">
      <rect>
       <x>560</x>
       <y>43</y>
       <width>221</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Kalameh Bold</family>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">#available
{

border-radius:25px;
color:#00204a;
}</string>
     </property>
     <property name="text">
      <string>Only available goods</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="search">
     <property name="geometry">
      <rect>
       <x>51</x>
       <y>41</y>
       <width>221</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Kalameh Bold</family>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">#search
{
background:#00bbf0;
border-radius:25px;
color:#00204a;
}</string>
     </property>
    </widget>
    <zorder>frame_2</zorder>
    <zorder>deleteproduct_lable_4</zorder>
    <zorder>frame_11</zorder>
    <zorder>name_4</zorder>
    <zorder>frame_6</zorder>
    <zorder>frame_10</zorder>
    <zorder>addtocart</zorder>
    <zorder>deleteproduct_lable_5</zorder>
    <zorder>count</zorder>
    <zorder>frame_7</zorder>
    <zorder>frame_9</zorder>
    <zorder>search_lable</zorder>
    <zorder>available</zorder>
    <zorder>search</zorder>
    <zorder>group</zorder>
   </widget>
   <widget class="QWidget" name="tab">
    <attribute name="title">
     <string>Cart</string>
    </attribute>
    <widget class="QFrame" name="frame_12">
     <property name="geometry">
      <rect>
       <x>450</x>
       <y>270</y>
       <width>311</width>
       <height>51</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">#frame_12{
background:#00bbf0;
border-radius:25px;
color:#00204a;
}</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QLabel" name="pricesum">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>10</y>
        <width>261</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Kalameh ExtraBold</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QLabel{
color:#005792}</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </widget>
    <widget class="QFrame" name="frame_13">
     <property name="geometry">
      <rect>
       <x>440</x>
       <y>80</y>
       <width>311</width>
       <height>51</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">#frame_13{
background:#00bbf0;
border-radius:25px;
color:#00204a;
}</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QLabel" name="delete_product">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>9</y>
        <width>231</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Kalameh ExtraBold</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QLabel{
color:#005792}</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="deleteButton">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>10</y>
        <width>41</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">#deleteButton{
background:#00bbf0;
border-radius:25px;
color:#00204a;
}</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>D:/grocery_store/grocery_store/assets/icons/TabIcon/trash.png</normaloff>D:/grocery_store/grocery_store/assets/icons/TabIcon/trash.png</iconset>
      </property>
     </widget>
    </widget>
    <widget class="QFrame" name="frame_3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>70</y>
       <width>415</width>
       <height>332</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">#frame_3{
background:#00bbf0;
border-radius:25px;
color:#d9faff}</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QTableView" name="product_list_cart">
      <property name="geometry">
       <rect>
        <x>13</x>
        <y>18</y>
        <width>389</width>
        <height>298</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Kalameh Bold</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">#product_list_cart{
background:#00bbf0;
border:noun;
color:#00204a}</string>
      </property>
      <attribute name="horizontalHeaderMinimumSectionSize">
       <number>100</number>
      </attribute>
      <attribute name="horizontalHeaderDefaultSectionSize">
       <number>100</number>
      </attribute>
     </widget>
    </widget>
    <widget class="QLabel" name="search_lable_2">
     <property name="geometry">
      <rect>
       <x>460</x>
       <y>50</y>
       <width>211</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Kalameh ExtraBold</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel{
color:#005792}</string>
     </property>
     <property name="text">
      <string>Choice to Delete</string>
     </property>
    </widget>
    <widget class="QLabel" name="search_lable_3">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>40</y>
       <width>101</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Kalameh ExtraBold</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel{
color:#005792}</string>
     </property>
     <property name="text">
      <string>Your cart</string>
     </property>
    </widget>
    <widget class="QPushButton" name="purchase">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>450</x>
       <y>340</y>
       <width>311</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Kalameh Bold</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">#purchase{
background:#005792;
border-radius:25px;
color:#d9faff;

}
#purchase:hover{
background:#00204a;
border-radius:25px;
color:#d9faff;

}</string>
     </property>
     <property name="text">
      <string>Purchase</string>
     </property>
    </widget>
    <widget class="QLabel" name="search_lable_4">
     <property name="geometry">
      <rect>
       <x>470</x>
       <y>240</y>
       <width>211</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Kalameh ExtraBold</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel{
color:#005792}</string>
     </property>
     <property name="text">
      <string>Sum Of Price</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="assets.qrc"/>
 </resources>
 <connections/>
</ui>
